coderadar-server/coderadar-core/src/main/java/org/wickedsource/coderadar/user/service/RegistrationService.java

```diff
@@ -54,10 +55,10 @@ public boolean userExists(UserRegistrationDataResource userRegistrationDataResou
    * @return User with id <code>userId</code>
    */
   public User getUser(Long userId) {
-    User user = userRepository.findOne(userId);
-    if (user == null) {
+    Optional<User> user = userRepository.findById(userId);
+    if (!user.isPresent()) {
       throw new ResourceNotFoundException();
```

Comment:
again, `ResourceNotFoundException` introduces a dependency to the rest layer. 
Perhaps move `ResourceNotFoundException` to the package `core.common` in order to avoid this dependency.

Reply:
I have moved the Exception to the `core.common` package.